netlify-plugin-cloudinary/src/index.ts

```diff
@@ -256,43 +256,54 @@ export async function onBuild({
     await Promise.all(
       CLOUDINARY_ASSET_DIRECTORIES.map(
         async ({ inputKey, path: defaultPath }) => {
-          let mediaPaths = inputs[inputKey as keyof Inputs] || defaultPath;
+          try {
+            let mediaPaths = inputs[inputKey as keyof Inputs] || defaultPath;
+
+            // Unsure how to type the above so that Inputs['privateCdn'] doesnt mess up types here
+
+            if ( !Array.isArray(mediaPaths) && typeof mediaPaths !== 'string' ) return;
+
+            if ( !Array.isArray(mediaPaths) ) {
+              mediaPaths = [mediaPaths];
+            }
+
+            mediaPaths.forEach(async mediaPath => {
+              const cldAssetPath = `/${path.join(PUBLIC_ASSET_PATH, mediaPath)}`;
+              const cldAssetUrl = `${host}${cldAssetPath}`;
+
+              const { cloudinaryUrl: assetRedirectUrl } = await getCloudinaryUrl({
```

Comment:
currently as it stands, you're failing the entire build for one upload error, didn't we agree to not fail the build but log it? as a developer, if you push out a build, would you prefer your build to fail for 1 failed image upload?